css/component/alert.css

```diff
@@ -0,0 +1,59 @@
+.alert{
+    padding:1rem;
+    border: 2px solid;
+    font-size: 1.25rem;
+}
+.primary-alert{
+    color: #004085;
+    background-color: #cce5ff;
+    border-color: #b8daff;
+
```

Comment:
You can use variables for colors instead of using hex values. That way you can reuse them anywhere in the code

Reply:
Yeah I am working on that, thank you for reviewing